### PR TITLE
Remove unnecessarily fully qualified names in test code. #1555

### PR DESCRIPTION
**Pull request by mkordas**
Fixes `UnnecessaryFullyQualifiedName` inspection violations in test code.

Description:
>Reports on fully qualified class names which can be shortened. The quick fix for this inspection will shorten the fully qualified names, adding import statements as necessary.

### CODE REVIEW

File: src/it/java/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/EmptyLineSeparatorTest.java [hunk 1]
```diff
@@ -24,7 +24,7 @@ public static void setConfigurationBuilder() throws CheckstyleException, IOExcep
     @Test
     public void emptyLineSeparatorTest() throws IOException, Exception {
         
-        java.lang.Class<EmptyLineSeparatorCheck> clazz = EmptyLineSeparatorCheck.class;
+        Class<EmptyLineSeparatorCheck> clazz = EmptyLineSeparatorCheck.class;
```
Comment (mkordas) [inline src/it/java/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/EmptyLineSeparatorTest.java]: @romani, do you know why compilation fails after removing `java.lang` here?
```
/home/travis/build/checkstyle/checkstyle/src/it/java/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/EmptyLineSeparatorTest.java:[27,14] type com.google.checkstyle.test.chapter4formatting.rule461verticalwhitespace.Class does not take parameters
```

Comment (romani) [inline src/it/java/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/EmptyLineSeparatorTest.java]: I am surprised too, can not do investigation from phone, willtry to take a close look later

Comment (romani) [inline src/it/java/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/EmptyLineSeparatorTest.java]: the same compilation error for java8 and java7 , when I move that code to separate stanalone class to compile it by "javac" it works fine.

```
+import java.lang.Class;
+

```
that is workaround for a problem, but importing "java.lang" most likely just another violation in Idea.

Comment (mkordas) [inline src/it/java/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/EmptyLineSeparatorTest.java]: OK, I'll try to isolate the problem and I'll ask the question on
StackOverflow.

Comment (romani) [inline src/it/java/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/EmptyLineSeparatorTest.java]: please separate that problem to separate PR, to avoid blocking other changes.

Comment (mkordas) [inline src/it/java/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/EmptyLineSeparatorTest.java]: Issue solved - we need to remember that Maven has different classpath than IDE - it's extended with compilable inputs - in this case `EmptyLineSeparatorInput.java`.

Comment (romani) [inline src/it/java/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/EmptyLineSeparatorTest.java]: Sounds like a bug in maven plugin, I do not see any reason it should be used in Check.

**Comment by romani** [comment]
merged as FF
